bundler/spec/dependabot/bundler/file_updater/gemfile_updater_spec.rb

```diff
@@ -231,11 +231,12 @@
       let(:requirements) do
```

Comment:
I think the `let(:dependency)` needs to be updated to `dependabot-test-ruby-package`? `version` possibly also changed to `1.1.0` and `previous_version` to `1.0.0`

Reply:
oh I think git sources need the commit or tag as the version